samples/notebooks/prediction/SDK_Pytorch_Custom_Predict.ipynb

```diff
@@ -0,0 +1,1603 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud auth login"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Grant roles to the service account. Since we will be using this service account to download the model artifacts from Google Cloud Storage, we grant the service account **Storage Object Admin** role. See [IAM roles for Cloud Storage](https://cloud.google.com/storage/docs/access-control/iam-roles) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud projects add-iam-policy-binding $PROJECT_ID \\\n",
+    "    --member=serviceAccount:{SERVICE_ACCOUNT}@{PROJECT_ID}.iam.gserviceaccount.com \\\n",
+    "    --role=roles/storage.objectAdmin"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Specify the service account key file name."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CREDENTIALS_FILE = \"./credentials.json\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Generate the service account key file. This will be used while running the custom serving container locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud iam service-accounts keys create $CREDENTIALS_FILE \\\n",
+    "    --iam-account=\"{SERVICE_ACCOUNT}@{PROJECT_ID}.iam.gserviceaccount.com\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pJe8TCr4ec4F"
+   },
+   "source": [
+    "#### Run and send requests to the container locally\n",
+    "\n",
+    "With the Custom Prediction Routine feature, it is easy to test the container locally.\n",
+    "\n",
+    "Note: You need to have the credentials set up in the previous step and pass the path to the credentials while running the container. The service account should have the **Storage Object Admin** permission.\n",
+    "\n",
+    "In this example, the container executes a prediction request and a health check."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "62ed2d334d0f"
+   },
+   "outputs": [],
+   "source": [
+    "with local_model.deploy_to_local_endpoint(\n",
```

Comment:
Is there a non-janky way I can use deploy_to_local_endpoint interactively? (i.e. without the context manager?) I want the container to hang out for a while so that I can time how long a request takes, and I don't want to block the notebook kernel. Actuating the context manager hooks directly works, but is not a good interface.

e.g. right now I'm doing
```
<cell>
endpt = local_model.deploy_to_local_endpoint(
    artifact_uri=f"{BUCKET_NAME}/{MODEL_ARTIFACT_DIR}",
    credential_path=CREDENTIALS_FILE,
)

local_endpoint = endpt.__enter__()
</cell>

<cell>
%%timeit
predict_response = local_endpoint.predict(
    request_file=INPUT_FILE,
    headers={"Content-Type": "application/json"},
)
</cell>
</cell>
endpt.__exit(None, None, None)
</cell>

Reply:
The above is the only way I'm aware of.

Reply:
OK, then please consider supporting this use case, either in this PR or a future one.

Reply:
Easiest thing to do would be to make some friendly aliases for local_endpoint's context manager methods.

Reply:
Ack.